Yasgui/packages/yasgui/src/index.ts

```diff
@@ -189,31 +189,50 @@ export class Yasgui extends EventEmitter {
         if (executeIdAfterInit && executeIdAfterInit === activeTabId) {
           (this.getTab(activeTabId) as Tab).query().catch(() => {});
         }
-        // }
       }
     }
   }
 
   public getHandleLocalStorageQuotaFull(): (e: any) => void {
     return () => {
-      this.resetResults();
+      this.reInitYasgui({resetYasr: true});
     };
   }
 
-  public resetResults(resetCurrentTab = false) {
+    /**
+     * Reinitializes the YASGUI component with default values from the config.
+     * @param {YasguiResetFlags} resetFlags object defining the flags for what to reset
+     * @param {YasguiConfiguration} config the YasguiConfiguration used to get the default values
+     */
+  public reInitYasgui(resetFlags: any, config: any = {}) {
```

Comment:
Why `any`? Doesn't these argument have types?

Reply:
I added a type containing what's used in the method